low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -670,6 +670,9 @@ def _get_nwalert(self, interfaces):
         try:
             for interface in interfaces:
                 interface_name = interface.get("ifId")
+                # If interface is eno2, skip processing it since it has very limited use right now

```

Comment:
Pre-pend LDR R1 
LDR R1 : Skip 'eno2' network interface monitoring as its unused, so no alerts should come for same

Reply:
done